### PR TITLE
Clean up text about notices

### PR DESCRIPTION
**Pull request by martinthomson**
This consolidates text about notices in the "communicating policies"
section, which seems most appropriate.

### CODE REVIEW

File: draft-ietf-git-using-github.md [hunk 1]
```diff
@@ -188,10 +188,24 @@ IETF tools be able to determine this and easily find the policy and GitHub
...
+interact with the GitHub repository.
+
+GitHub prominently links to the CONTRIBUTING file on certain pages.  This file
+SHOULD be used in preference to the README for information that new contributors
+need.  A link to the CONTRIBUTING file from the README is advised.
+
+In addition to Working Group policies, notices on repositories MUST include
+citations for BCPs 78 and 79 ({{!COPYRIGHT=RFC5378}}, {{!IPR=RFC8179}}) and
+other IETF contribution policies.  A copy or reference to the [IETF Note
+Well](https://www.ietf.org/about/note-well/) is appropriate.
```
Comment (richsalz) [inline draft-ietf-git-using-github.md]: also appropriate?  Or do you mean this is an appropriate way to meet the 78/79 requirements?

File: draft-ietf-git-using-github.md [hunk 2]
```diff
@@ -188,10 +188,24 @@ IETF tools be able to determine this and easily find the policy and GitHub
...
+recorded in a stable location other than the mailing list archive.  This also
+makes Working Group policies available to casual contributors who might only
+interact with the GitHub repository.
+
+GitHub prominently links to the CONTRIBUTING file on certain pages.  This file
+SHOULD be used in preference to the README for information that new contributors
+need.  A link to the CONTRIBUTING file from the README is advised.
+
+In addition to Working Group policies, notices on repositories MUST include
+citations for BCPs 78 and 79 ({{!COPYRIGHT=RFC5378}}, {{!IPR=RFC8179}}) and
```
Comment (DavidSchinazi) [inline draft-ietf-git-using-github.md]: Could we instead just mandate a reference to the note well? I think the IETF Code of Conduct and Anti-Harassment Procedures are at least as import as Copyrights and IPR, and singling out some parts of the Note Well doesn't make sense to me.

Comment (chris-wood) [reply draft-ietf-git-using-github.md]: I like this suggestion!

Comment (martinthomson) [reply draft-ietf-git-using-github.md]: I agree.  It also simplifies this text more, so double win.

File: draft-ietf-git-using-github.md [hunk 3]
```diff
@@ -256,6 +248,14 @@ editors are able to do in order to serve these goals.  Chairs SHOULD
 periodically consult with document editors to ensure that policies are effective
 and not unjustifiably constraining progress.
 
+A document editor can still use GitHub independently for documents that they
+edit, even if the Working Group does not expressly choose to use GitHub.  Any
+such public repository MUST follow the guidelines in BCPs 78 and 79
+({{!COPYRIGHT=RFC5378}}, {{!IPR=RFC8179}}) and bear notices to that effect.
```
Comment (DavidSchinazi) [inline draft-ietf-git-using-github.md]: can we also replace these with the note well?

Comment (martinthomson) [reply draft-ietf-git-using-github.md]: I honestly don't know.

Comment (DavidSchinazi) [reply draft-ietf-git-using-github.md]: In that case, can we require a link to the Note Well on top of what's already required?

**Review by DavidSchinazi — APPROVED** [review]
Thanks!

**Pull request merged**